governance/polls/templates/MIPs/Short Ratification Poll - $mip_code - $date_MONTH_DD,_YYYY.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Short Ratification Poll for $proposal_name ($mip_code) - $date_MONTH_DD,_YYYY
+summary: $sentence_summary
+discussion_link: $discussion_link
+vote_type: Plurality Voting
+categories:
+   - MIPs
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: $YYYY-MM-DDT16:00:00
+end_date: $YYYY-MM-DDT16:00:00
+---
+# Ratification Poll for $proposal_name ($mip_code) - $date_MONTH_DD,_YYYY
+
+The Governance Facilitators have placed a ratification poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP16](https://mips.makerdao.com/mips/details/MIP16). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for three days beginning on $date_DAY,_MONTH_DD at 16:00 UTC.
+
+As per $process_component, this subproposal meets the requirements to be entered into the Weekly Governance Cycle.
+
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to $mip_code being accepted and implemented in the Maker Protocol.
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*$paragraph_summary*
+
+Please review the links below to inform your position on this proposal before voting.
+* [Canonical Proposal Version]($link_to_github_commit_version)
+* [Latest Proposal Version]($link_to_portal_version)
+* [Proposal Discussion Thread]($discussion_link)
+
+## Outcomes
+
+This poll implements a **Minimum Positive Participation** value. The Minimum Positive Participation is currently set to **10,000 MKR**.
+
+**If the votes for the 'Yes' option exceed the votes for the 'No' option AND the votes for the 'Yes' option exceed 10,000 MKR, then the following actions will be taken:**
+* The MIP Editors will mark the proposal **Accepted** and the Governance Facilitators will confirm its passage on the Governance and Risk call on $upcoming_governance_call_date. 
+* Any further work required to implement the proposal will be tasked to the relevant [Core Units](https://mips.makerdao.com/mips/details/MIP38#mip38c2-core-unit-state).
+
+**Otherwise, this proposal will be marked as rejected**
```

Comment:
Note that this differs from the monthly-cycle template, where: "... as rejected per MIP51". Not convinced that MIP16 is a perfect fit here.

Reply:
Yeah, we would need to update MIP16 first, as it doesn't talk about this at all. I think that section needs to be parameterized, so it can link to whichever MIP is allowed the short ratification poll.